src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getAllCustomersValidator = () => {
+    return celebrate({
+      [Segments.QUERY]: Joi.object().keys({
+        offset: Joi.string(),
+        limit: Joi.string(),
```

Comment:
Mesma coisa, o campo `limit` poderia ser um número inteiro aqui, e com valor padrão de 50 caso o usuário não informasse.

```suggestion
        limit: Joi
            .number()
            .integer()
            .positive()
            .max(50)
            .default(50),
```